### ISSUE TITLE
Styleguide fails with "Internal Server Error"

### ISSUE THREAD

**Issue by apkoponen** [HSLdevcom/digitransit-ui#669]
https://beta.reittiopas.fi/styleguide/ fails with "Internal Server Error".

From my local dev the reason is:

```
ReferenceError: document is not defined
   at StyleGuidelinesPage.getIcons (StyleGuidelines.js:220:33)
```

**Comment by Macroz** [comment]
This should be fixed in one of the branches already, so waiting for merging.

**Comment by hukka** [comment]
Should be fixed in #661. Reopen if problem persists.
